docs/PLUGIN_CATALOG.md

```diff
@@ -137,6 +135,5 @@ There will always be more plugins we wish we had. To make sure others can contri
 | [#1196](https://github.com/intelsdi-x/snap/issues/1196) | publisher | Plugin Wanted: Publisher for Prometheus  |
```

Comment:
Publisher for Prometheus should not be longer on the wishlist - I closed the issue "Plugin Wanted: Publisher for Prometheus", so @mbbroberg, could you run plugin-sync again?